purchases/src/defaults/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -102,7 +102,8 @@ class Purchases internal constructor(
      *
      * @param [listener] Called when all purchases have been synced with the backend, either successfully or with
      * an error. If no purchases are present, the success function will be called.
-     * @warning This function should only be called if you're migrating to RevenueCat or in observer mode.
+     * @warning This function should only be called if you're migrating to RevenueCat or if you have instructed
+     * `Purchases` not to [finish transactions][finishTransactions].
```

Comment:
Is `[finish transactions][finishTransactions]` special syntax for a link or something?

Reply:
Yea! The standard link is `[<link>]`, but you can change the "anchor text" by adding a second set of square brackets in front of it, so `[<anchor>][<link>]`. Shows up like this in the docs: 
![image](https://github.com/RevenueCat/purchases-android/assets/29483617/ac44c5fa-d65d-47c9-840c-fdf839b4abb6)